### PR TITLE
30 separate travis runs php output redirected output of php to /dev/null

### PR DESCRIPTION
**Pull request by level420**
A lot of tests do not clean up instances and other garbage at test end.
This makes other tests fail, even if calling the testclasses one by one
does not fail. We try to split up the runs to avoid these interferences.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,35 @@ before_script:
   - export PATH=$PWD/phantomjs-2.1.1-linux-x86_64/bin/:$PATH
 script:
   - tool/test/travis/setup
-  - php -S 127.0.0.1:31323 1>&2 &
-  - tool/test/travis/run http://127.0.0.1:31323/framework/test-console/index-source.html
+  - php -S 127.0.0.1:31323 1>/dev/null 2>/dev/null &
+  - tool/test/travis/run "http://127.0.0.1:31323/framework/test-console/index-source.html?testclass=qx.test.application"
```
Comment (oetiker) [inline .travis.yml]: the classic way to write this is:

```sh
php -S 127.0.0.1:31323 1>/dev/null 2>&1 &
```

**Comment by oetiker** [comment]
```
tool/test/travis/run "http://127.0.0.1:31323/framework/test-console/index-source.html?testclass=qx.test.dev
```

unfortunately, if it is done in this way, we can not see how many tests pass.

maybe altering the runner code in phantom to start a new 'browser object' per test would be a good solution.

**Comment by level420** [comment]
@oetiker do you mean that travis does not report success or failure for each "tool/test/travis/run ....." call?

**Comment by level420** [comment]
Giving up on this, as the test now hangs on tool/test/travis/run "http://127.0.0.1:31323/framework/test-console/index-source.html?testclass=qx.test.application".

Maybe someone else with more travis insight can have a look into this  ( @oetiker @cajus )

**Pull request closed without merging**